### PR TITLE
Fix dependencies

### PR DESCRIPTION
**Pull request by isht3**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -32,14 +32,20 @@
...
-        'colorama',
-        'pandas',
-        'requests',
-        'responses',
-        'validators',
+        'beautifulsoup4==4.6.0',
+        'beautifultable==0.5.0',
+        'click==6.7',
+        'coverage==4.5.1',
+        'coveralls==1.3.0',
```
Comment (RishabhJain2018) [inline setup.py]: Why do we need `coverage` and `coveralls` in requirements? Any specific reason?

File: setup.py [hunk 2]
```diff
@@ -32,14 +32,20 @@
...
+        'flake8==3.0.4',
+        'lxml==4.2.1',
+        'pytest==3.5.1',
+        'pytest-cov==2.5.1',
+        'python-dateutil==2.7.3',
+        'requests==2.18.4',
+        'requests-toolbelt==0.8.0',
```
Comment (RishabhJain2018) [inline setup.py]: Are you sure we need `requests-toolbelt`?

File: setup.py [hunk 3]
```diff
@@ -32,14 +32,20 @@
...
-        'validators',
+        'beautifulsoup4==4.6.0',
+        'beautifultable==0.5.0',
+        'click==6.7',
+        'coverage==4.5.1',
+        'coveralls==1.3.0',
+        'flake8==3.0.4',
+        'lxml==4.2.1',
+        'pytest==3.5.1',
+        'pytest-cov==2.5.1',
```
Comment (RishabhJain2018) [inline setup.py]: Are you sure we need `pytest-cov`?

File: setup.py (hunk 1 above)
Comment (isht3) [reply setup.py]: Sorry, I copied this from requirements. I'll change it.

File: setup.py (hunk 2 above)
Comment (isht3) [reply setup.py]: This will be installed as requests will be installed. So Ill change this

**Pull request merged**